src/accordion/types.ts

```diff
@@ -12,6 +12,9 @@ export type AccordionItemType = "default" | "small";
 export interface AccordionItemProps {
     title: string;
     children: JSX.Element | JSX.Element[];
+    /**
+     * Only for initial state, do not use to programmatically expand/collapse Accordion.Item after first load
```

Comment:
The `expanded` prop is used for controlling the states for each item though. It is not restricted to the initial load only